Contest_Package.md

```diff
@@ -40,37 +40,35 @@ Information in the API is always either in JSON format,
 - The NDJSON returned from the endpoint `/contests/<id>/<endpoint>` is stored as
   `<endpoint>.ndjson`. (The only such endpoint is `event-feed`.)
 
-
-Files referenced in `api.json` and `contest.json` are stored as `api/<filename>`
-and `contest/<filename>` respectively, and files referenced in
-`<endpoint>.json` are stored as `<endpoint>/<id>/<filename>`, where:
-
-- `<id>` is the ID of the endpoint object the reference is in.
-- `<filename>` is the filename specified in the file reference object.
-
 Some of these API endpoints are often written by humans in a package. For this
```

Comment:
I realize this was not changed in the PR, but "these API endpoints are often written by humans in a package" makes no sense?

Reply:
I've rewritten it. Hopefully it's clearer now.

Reply:
I think it's clearer now.